Hcal/include/Hcal/VisiblesCutflow.h

```diff
@@ -0,0 +1,48 @@
+#ifndef HCAL_VISIBLESCUTFLOW_H_
+#define HCAL_VISIBLESCUTFLOW_H_
+
+// LDMX
+#include "Framework/Configure/Parameters.h"
+#include "Framework/EventProcessor.h"
+#include "Tools/ONNXRuntime.h"
+
+namespace hcal {
+
+class VisiblesCutflow : public framework::Analyzer {
+ public:
+  VisiblesCutflow(const std::string& name, framework::Process& process)
+      : Analyzer(name, process) {}
+
+  ~VisiblesCutflow() override = default;
+
+  void configure(framework::config::Parameters& parameters) override;
+
+  void analyze(const framework::Event& event) override;
+
+ private:
+  std::unique_ptr<ldmx::Ort::ONNXRuntime> rt_;
+  double bdtCutVal_{0.};
```

Comment:
```suggestion
  double bdt_cut_val_{0.};
```
again naming in this file too